addon/components/tooltip-on-element.js

```diff
@@ -430,6 +440,16 @@ export default EmberTetherComponent.extend({
     this.sendAction('onTooltipShow', this);
   },
 
+  stopTether() {
+    run.schedule('afterRender', this, function() {
+      this.get('_tether').disable();
+    });
+  },
+
+  startTether() {
+    this.get('_tether').enable();
+  },
```

Comment:
We copied your fix here at Zenefits and did this to get test coverage

https://github.com/zenefits/ember-tooltips/pull/21/files#diff-eafc17148ee89489b3d8098122b506e6R409